tc-build.sh

```diff
@@ -4,6 +4,7 @@ set -xe
 
 source ${HOME}/DeepSpeech/tf/tc-vars.sh
 
+TFDIR=${HOME}/DeepSpeech/tf
```

Comment:
I would prefer we name that differently from one of the uses below. For example, `DS_TF_DIR` :)